__tests__/app.test.js

```diff
@@ -213,6 +213,7 @@ describe("GET /api/articles with comment count", () => {
         expect(articles).toBeSorted({ key: "created_at", descending: true });
       });
   });
+
   test("Bad request", () => {
```

Comment:
what is this test doing?

Reply:
This is checking for default sorted and order. By default, the order is desc and sorted by date.

Reply:
Added the test cases now.